frontend/src/App.tsx

```diff
@@ -11,6 +11,23 @@ import CustomParseFormat from 'dayjs/plugin/customParseFormat'
 import RelativeTime from 'dayjs/plugin/relativeTime'
 import { OfflineStatus } from './components/OfflineStatus'
 import { NotificationBar } from './components/Supper/NotificationBar'
+import { APIEntry } from './store/facilityBooking/types'
+import BookingBlock from './components/ScheduleBlock/BookingBlock'
+import ViewScheduleBlock from './components/ScheduleBlock/ViewScheduleBlock'
+import ViewMyBookings from './routes/FacilityBooking/MyBookings'
+
+const mockValues: APIEntry[] = [
+  {
+    bookingID: 2,
+    ccaID: 1,
+    description: 'Lol',
+    endTime: 1615924800,
+    eventName: 'Lol',
+    facilityID: 1,
+    startTime: 1615910400,
+    userID: 'A0165780X',
```

Comment:
good to see my matric number living on HAHAHAHA

Reply:
You can never leave